coursebuilder/common/utils.py

```diff
@@ -339,3 +340,33 @@ def find_youtube_video_id(text):
         if match:
             return match.group(1)
     return None
+
+
+def embed_carousel(content):
+    start_key = '[start_carousel]'
+    stop_key = '[stop_carousel]'
+
+    while True:
```

Comment:
@andela-angene Can we use pattern matching to validate even before this step?

Reply:
@chegejames do you mean I should check if the keyword(s) exists before starting the loop?

Reply:
@andela-angene You can save a bit of time. Just try a simple regex.

Reply:
Updated, the tests are still passing.